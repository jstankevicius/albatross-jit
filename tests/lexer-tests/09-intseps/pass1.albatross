1_000 1_000_000
1000 1000000
