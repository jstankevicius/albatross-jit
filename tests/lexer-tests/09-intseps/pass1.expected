1 1 INT 1000
7 1 INT 1000000
1 2 INT 1000
6 2 INT 1000000
